lib/connectHapiServerToRedis.js

```diff
@@ -2,7 +2,66 @@
 // Then run a callback (optional)
 // The node_redis redis client object instance will be stored here: server.plugins['hapi-redis'].client
 
+// PLUGIN code - consider moving elsewhere if necessary
+
+var authPlug = {
+  register: function(plugin, options, next) {
+    plugin.auth.scheme('seelio_auth', function( server, options) {
+      return {
+        authenticate: function(request, reply) {
+          options.authenticate(request, function(err, credentialsObj) {
+            if (err) {
+              return reply(err, null, {});
+            }
+            else {
+              return reply.continue({credentials: credentialsObj});
+            }
+          });
+        }
+      }
+    });
```

Comment:
This return statement is sorta messy. Maybe something like this would be more readable.

plugin.auth.scheme('seelio_auth', function( server, options) {
 + var authentication = function(request, reply) {
+          options.authenticate(request, function(err, credentialsObj) {
+            if (err) {
+              return reply(err, null, {});
+            }
+            else {
+              return reply.continue({credentials: credentialsObj});
+            }
+          });
+        }
+      }
+      return {
+        authenticate: authenticate
+      }
+    });